examples/integration_test.py

```diff
@@ -13,24 +13,26 @@
 from aptos_sdk.account_address import AccountAddress
 from aptos_sdk.aptos_cli_wrapper import AptosCLIWrapper, AptosInstance
 
-from .common import APTOS_CORE_PATH
-
 
 class Test(unittest.IsolatedAsyncioTestCase):
     _node: Optional[AptosInstance] = None
+    _aptos_core_path: str
 
     @classmethod
-    def setUpClass(self):
+    def setUpClass(cls):
+        cls._aptos_core_path = os.getenv("APTOS_CORE_PATH")
+        if not cls._aptos_core_path:
+            raise Exception("Environment variable `APTOS_CORE_PATH` is not set")
+
         if os.getenv("APTOS_TEST_USE_EXISTING_NETWORK"):
             return
 
-        self._node = AptosCLIWrapper.start_node()
-        operational = asyncio.run(self._node.wait_until_operational())
+        cls._node = AptosCLIWrapper.start_node()
+        operational = asyncio.run(cls._node.wait_until_operational())
         if not operational:
-            raise Exception("".join(self._node.errors()))
+            raise Exception("".join(cls._node.errors()))
 
         os.environ["APTOS_FAUCET_URL"] = "http://127.0.0.1:8081"
-        os.environ["APTOS_INDEXER_CLIENT"] = "none"
```

Comment:
so what you're saying is that this doesn't actually set the environment variable and it is overwritten each time in common.py?

That doesn't make a lot of sense according to the docs.

The way it should work is that we set this once during the class call to "none", each common.py should see that it is set to "none" and leave it alone, during loading of the indexer client we should see it is "none" and not touch it.